_sass/core/global.scss

```diff
@@ -229,26 +228,6 @@ table {
   }
 }
 
-.admonitionblock.note, .note, .admonitionblock.tip, .tip {
-  tbody .icon {
-    background-color: $teal;
-  }
-  tbody .content {
-    background-color: $blue;
-  }
-}
-
-.admonitionblock {
-  tbody .icon {
-    text-align: center;
-    width: 50px;
-  }
-}
-
-.admonitionblock + .admonitionblock {
-  margin-top: 20px;
-}
```

Comment:
This was added to work around various issues with our spacing model. It's all useless now that we rely on the Asciidoctor spacing model.